evaluator/src/main/kotlin/ProjectSourceRule.kt

```diff
@@ -34,7 +33,7 @@ import org.ossreviewtoolkit.utils.common.FileMatcher
 open class ProjectSourceRule(
     ruleSet: RuleSet,
     name: String,
-    projectSourceResolver: SourceTreeResolver = ruleSet.ortResult.createResolver()
+    projectSourceResolver: SourceTreeResolver = ruleSet.projectSourceResolver
```

Comment:
Is there now a reason to keep the `projectSourceResolver` constructor parameter instead of accessing `ruleSet.projectSourceResolver` directly? If it was for testability, then `ProjectSourceRuleTest` could also either use the `RuleSet` constructor directly instead of the `ruleSet` helper function, or extend the latter to also accept a `SourceTreeResolver`.

Reply:
I've implemented your 2nd proposal.